.circleci/config.yml

```diff
@@ -1,75 +1,105 @@
 version: 2
-executorType: machine
-stages:
-  build:
-    workDir: ~/electricitymap
+jobs:
+  lint_python:
+    docker:
+      - image: circleci/python:3.6.3
     steps:
-      - type: checkout
-      - type: shell
-        name: Set Python version
-        command: |
-          set -eux -o pipefail
-          pyenv install --list
-          pyenv install 3.6.3
-          pyenv global 3.6.3
-      - type: shell
-        name: Upgrade pip
-        command: pip install -q --upgrade pip
-      - type: shell
-        name: Install Docker Compose
-        command: pip install docker-compose==1.24.1 --progress-bar=off
-      - type: shell
-        name: Show versions
-        command: |
-          docker --version
-          docker-compose --version
+      - checkout
       - restore_cache:
           key: pip-{{ .Branch }}-{{ checksum "parsers/requirements.txt" }}
-      - type: shell
-        name: Lint
-        command: |
-          set -eux -o pipefail
-          python --version
-          pip -v --version
-          pyenv versions
-          pip install -q flake8 pylint==2.3.1 --progress-bar=off
-          sudo apt-get install libxml2-dev
-          pip install -q -r parsers/requirements.txt --progress-bar=off
-          # stop the build if there are Python syntax errors or undefined names
-          flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
-          # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
-          # flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-          pylint -E parsers/*.py -d unsubscriptable-object,unsupported-assignment-operation,unpacking-non-sequence
-          python -m unittest discover parsers/test
-          npm install -g jsonlint
-          jsonlint -q config/*.json web/locales/*.json
+      - run:
+          name: Upgrade pip
+          command: sudo pip install -q --upgrade pip
```

Comment:
How come `sudo` is necessary? Also for `npm install` below..

Reply:
Circle CI requirements.. It's the only way I could make it work. `--user` didn't cut it

Reply:
Alright, thanks for the explanation! :+1: